src/components/Products/Products.tsx

```diff
@@ -0,0 +1,46 @@
+import "./Products.css";
+
+import React, { useEffect, useState } from "react";
+
+interface ProductParams {
+  id: string;
+  title: string;
+  category: string;
+  price: string;
+  description: string;
+  image: string;
+}
+
+const Products = () => {
+  const [products, setProducts] = useState<ProductParams[]>([]);
+
+  useEffect(() => {
+    fetch("https://fakestoreapi.com/products")
+      .then(res => res.json())
+      .then(json => setProducts(json));
+  }, []);
+
+  return (
+    <div className="main-wrapper" data-testid="products-page">
+      {products.map((product: ProductParams) => (
+        <Product key={product.id} {...product} />
+      ))}
+    </div>
+  );
+};
+
+export default Products;
+
+const Product = (params: ProductParams) => {
```

Comment:
Create a separate component for Product. Its good practice to have one component per file.

Reply:
addressed the comment.